tldw_Server_API/tests/Claims/test_claims_utils_persistence.py

```
@@ -0,0 +1,106 @@
+import contextlib
+
+import pytest
+
+from tldw_Server_API.app.core.Claims_Extraction import claims_utils
+
+
+pytestmark = pytest.mark.unit
+
+
+class _InlineLoop:
+    async def run_in_executor(self, _executor, func):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `run_in_executor`

(ANN202)